server/routes/referrals/riskInformationPresenter.ts

```diff
@@ -32,16 +32,16 @@ export default class RiskInformationPresenter {
   }
 
   get additionalRiskInformation(): Record<string, string | null> {
-    return this.riskPresenter.riskSummaryEnabled
+    return this.riskPresenter.riskSummaryNotFound
       ? {
-          label: 'Additional risk information',
```

Comment:
the changes here are swapping the order of the ternary, `since riskSummaryEnabled ~== !riskSummaryNotFound`